integration_tests/tests/test_dbt_artifacts/test_artifacts.py

```diff
@@ -19,6 +19,11 @@ def test_artifacts_caching(dbt_project: DbtProject):
     second_row = read_model_artifact_row(dbt_project)
     assert first_row == second_row, "Artifacts are not cached at the on-run-end."
 
+
+def test_artifacts_updating(dbt_project: DbtProject):
```

Comment:
Separated the original test to two, with the second test also running for dbt 1.3.0